mac

```diff
@@ -139,12 +139,11 @@ brew "imagemagick"
 # Testing
 brew "qt@5.5"
 
-# Programming languages and package managers
+# Programming language prereqs and package managers
 brew "libyaml" # should come after openssl
-brew "node"
-brew "rbenv"
-brew "ruby-build"
+brew "coreutils"
 brew "yarn"
+cask "gpgtools"
```

Comment:
Coreutils is a requirement of one of the plugins. 

The node plugin has a feature where it verifies the release signature as being from someone on the Node release team. It uses pgp for this. Before installing node, you see I run a script asdf provides to import the release team members keys. 

We could skip this by setting an env variable before install but it will come up on every node install after this as well. So it seemed best to go ahead and gpg. 

Gpgtools cask includes the UI components that will be more friendly for most users to interact with - GUI pin entry, GUI keychain, etc - while still also allowing command line access. Seemed like the best choice for laptop.